draft-ietf-rats-corim.md

```diff
@@ -1591,129 +1591,118 @@ The Verifier SHOULD set the `authorized-by` field in Accepted Claims Set entries
-match the Attester. Tags/triples which match are accepted, and the Accepted Claims Set
-is extended using Endorsements etc. from the accepted tags.
+In the Accepted Claims Set extension phase, a CoRIM Appraisal Context and an Evidence Appraisal Policy are used by the Verifier to find CoMID tags or triples which match the ACS.
+Tags/triples which match are accepted, and the Accepted Claims Set is extended using Endorsements etc. from the accepted tags.
 
-[^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/136
+Each triple is processed independently of other triples.
+If any triple within the CoRIM Appraisal Context does not match the Accepted Claims Set then ACS extensions in that triple are not processed.
+If a triple does not match the ACS then this does not affect whether the Verifier tries to match other triples against the ACS.
 
-### Comparing and processing CoMID tags or triples
+Note that the order of triples processing is important. Processing a triple may result in modifications to the ACS which allow other triples to match the modified ACS.
+The Verifier SHOULD ensure that it processes triples in the correct order (for example by processing all triples which match against a particular ACS entry or environment after that ACS entry has been modified).
 
-[^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/136
+### Processing Reference Values Triple
 
-### Matching Evidence against Reference Values
+A Reference Value Provider (RVP) may publish Reference Values, which are reviewed to determine if they are contained in the Accepted Claims Set (ACS).
+The authority of the RVP is added to all ACS entries which match the Reference Values triple.
 
-An Endorser may use CoMID tags to publish Conditional Endorsements, which
-are added to the Accepted Claims Set only if specified conditions apply.
-This section describes the process performed by the Verifier to determine
-which Conditional Endorsements from the candidate CoMIDs should be added
-to the Accepted Claims Set.
+For each Reference Value triple, the verifier compares each `reference-triple-record` against ACS entries as described in {{sec-match-one-se}}, using a `stateful-environment-record` created from the fields in `reference-triple-record`.
+If all `reference-triple-record`s match the ACS then the authority which signed the triple shall be added to the `authorized-by` field of each ACS entry which matched.
 
-The verifier checks whether Conditional Endorsements are applicable by
-comparing Evidence in the Accepted Claims Set against Reference Values
-from the CoMID. These Reference Values may be provided as Reference Value
-Triples or may be combined with the Endorsements, for example as the
-Conditional Endorsement Series Triple.
+TODO: Is this really what we want? I think this description might not work as expected if there are partial matches.
```

Comment:
@andrew-draper To my impression: This PR is still as Work In progress, if I read Line 1614: